recipes/rapid-pe/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set name = "rapid-pe" %}
+{% set version = "0.0.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/rapid_pe-{{ version }}.tar.gz
+  sha256: 53739bd328779f8484cfc1887ba18aa7b8a0b0ea82b064bb4866b2cbc6eaa518
+
+build:
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+  skip: true  # [win]
```

Comment:
If you are skipping Windows just b/c the lack of pytorch then you should still make this package `noarch` and let the solver fail on Windows. Then when pytorch is available on Windows, this package will be ready.

Reply:
It's not PyTorch, but rather `lal` which doesn't support Windows (see [here](https://github.com/conda-forge/lal-feedstock/blob/df3c97bc502f2d24e1b3fdf8f4ef9f393abbd757/recipe/meta.yaml#L37)).    I simply copied the `skip` line from `lal`'s `meta.yaml`, though I'm happy to just let the pipeline fail if that's the preferred approach.

Also it looks like the executables would not be made accessible on Windows conda/conda-build#4385, which would make our package not very useful.

Reply:
`lal` is also unavailable on Windows but that one cannot be noarch, this one can. So the situation is the same. However, if this package uses the old and depreacated scripts, then you are correct, this cannot be noarch :-/